frontend/package.json

```diff
@@ -21,13 +21,15 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@mui/base": "^5.0.0-beta.40",
-    "@mui/icons-material": "^5.15.20",
+    "@mui/icons-material": "^5.17.1",
     "@mui/joy": "^5.0.0-beta.36",
-    "@mui/material": "^5.15.20",
+    "@mui/material": "^5.17.1",
     "@rollup/plugin-replace": "^5.0.7",
-    "@testing-library/jest-dom": "^6.1.3",
-    "@testing-library/react": "^15.0.5",
-    "@testing-library/user-event": "^14.5.1",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.3.0",
+    "@testing-library/user-event": "^14.6.1",
+    "@vitejs/plugin-react": "^4.4.1",
```

Comment:
Do these need to be upgraded as well?  If so, it's because the new version of React requires this, yes?

Reply:
yes, react 19 requires these packages to be upgraded